src/InventoryManager.cpp

```diff
@@ -78,3 +79,68 @@ int InventoryManager::userInput()
     
 	return 0;
 }
+
+/*uses ActiveInventory functions to create items from a csv file*/
+void InventoryManager::readCSVFile(const std::string &file)
+{
+	std::string name, str_id, cat, sub_cat, qty, sale_price;
+	std::string tax, total_price, buy_cost, profit, exp, tmp_line; 
+	unsigned long id;
+	
+	std::ifstream csv_file(file);
+
+	if(!csv_file.is_open()) std::cout << "ERROR: File Open" << '\n';
+	
+	/* This skips the first line for you. */
+	std::getline(csv_file, tmp_line);
+
+	while(csv_file.good()){
+
+		/*reading in the csv info, converting types*/
+		getline(csv_file, name, ',');
+
+		/*exit the loop if we domaken't read in a new name.*/
+		if (!csv_file.good()) break;
+
+		getline(csv_file, str_id, ',');
+		id = std::stoul(str_id);
+		getline(csv_file, cat, ',');
+		getline(csv_file, sub_cat, ',');
+		getline(csv_file, qty, ',');
+		getline(csv_file, sale_price, ',');
+		getline(csv_file, tax, ',');
+		getline(csv_file, total_price, ',');
+		getline(csv_file, buy_cost, ',');
+		getline(csv_file, profit, ',');
+		getline(csv_file, exp, '\n');
+
```

Comment:
It might work better to use fscanf here since we know the exact format and it would condense this down to 1 line instead of 11. It doesn't need to change, just a suggestion.

Reply:
i might go back and change this in this future or you are free to! but for now, i am happy with getting all the lines redundantly.

Reply:
We can't actually use fscanf here, anyways; you'd have to read it into a string and use sscanf, since we're using ```std::ifstream``` for input and not a ```FILE*``` which is what fscanf takes, and there's no way to convert it to that either.

Nonetheless, reading it into a string and then using sscanf would still reduce the line count.  But I think this is fine, I don't really care about line count and typically doing things the C++ way is safer.